aws-android-sdk-s3-test/src/androidTest/java/com/amazonaws/services/s3/AclIntegrationTest.java

```diff
@@ -144,8 +143,6 @@ public void testCannedAcls() throws Exception {
         for (int index = 0; index < accessControls.length; index++) {
```

Comment:
What is this test enforcing now? That only assert is checking bucket owner, which seems far removed from its original intent.